plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -149,7 +150,19 @@ func findNodesThatFit(
 	nodeNameToInfo map[string]*schedulercache.NodeInfo,
 	nodes []*api.Node,
 	predicateFuncs map[string]algorithm.FitPredicate,
-	extenders []algorithm.SchedulerExtender) ([]*api.Node, FailedPredicateMap, error) {
+	extenders []algorithm.SchedulerExtender,
+	metaFactory interface{},
+) ([]*api.Node, FailedPredicateMap, error) {
+
+	/**
+	 * PredicateMetadata is an internal package of predicates.  We don't want or need to know the type.
```

Comment:
I think i will delete this comment, since the function return is what is really important